project/MicroService.scala

```diff
@@ -50,6 +50,7 @@ trait MicroService {
     .settings(defaultSettings(): _*)
     .settings(
       routesGenerator := StaticRoutesGenerator,
+      routesImport += "controllers.Binders._",
```

Comment:
I was doing a similar thing in our current story with binders but yours will be merged first because we need your changes, so I will take your implementation.